client/src/components/AddNewTaskBtn/index.js

```diff
@@ -0,0 +1,19 @@
+import styles from './style.module.css'
+
+const AddNewTaskBtn = () => {
+
+    const handleClick = () => {
+        console.log('Add New Task Button clicked');
+    }
+
+    return (
+        <button 
+            className={styles.button} 
+            type='button'
```

Comment:
I'm curious here: is there a particular reason you added the type of button here? I think that given that you are correctly using the native element, we _shouldn't_ need to add the type. 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#best_practices

Reply:
Thanks for the feedback, looks like it doesn't need to be there.